### ISSUE TITLE
Confused about which command invokes interpolation and inserts the frame

### ISSUE THREAD

**Issue by deama** [MortenHannemose/pytorch-vfi-cft#11]
So I've made a way to compare scene cuts, however, I'm having some trouble trying to figure out how to simply duplicate a frame instead of interpolate it. I have an if statement which compares the current frame to a frame that happens just before a scene is cut, but I am not sure how I could just make it skip interpolation for that frame and instead just instert the same frame.

Any ideas?

**Comment by MortenHannemose** [comment]
You can add the comparison of the frames to `compute_inbetween_frames`. If there is a cut, you could call a special version of `recursive_predict_and_write`, where you make it duplicate the frames instead.

**Comment by deama** [comment]
Alright, so I tried some stuff but can't quite get it to work, here's the code:


[slow_movie.txt](https://github.com/MortenHannemose/pytorch-vfi-cft/files/3989343/slow_movie.txt) (rename to .py)


Essentially I use scenedetect python module, run it, it generates two images per scene, first one is after a cut, second one is just before a cut. The idea I have is to compare those two to whatever image is being processed, then simply skip it's interpolation and write the original instead.

The problem I'm having is for some reason, when it starts it works fine, the comparison against the first cut is done, skipped, a dupped frame is written, it's fine. But once it starts to compare the 2nd cut of the first scene, the if statement comparison just keeps on accepting the later frames even though they're not the same? I made a seperate test.py and manually compared the frames to the cuts to make sure the comparison was correct, and it was.

So I'm not quite sure what's happening, any ideas?
You basically just have to pip install scenedetect, run the command I have in the comments, it will generate a "cuts" file with all the scene cuts. I originally automated this, but commented it out for now because then it would keep deleting and remaking the cuts which would take like 3 minutes every run.

Oh and I also used the glob module to more easily get the files from the cuts folder.
